tests/library/Respect/Validation/Rules/HexRgbColorTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace Respect\Validation\Rules;
+
+class HexRgbColorTest extends \PHPUnit_Framework_TestCase
+{
+    public function testHexRgbColorValuesONLYShouldReturnTrue()
+    {
+        $validator = new HexRgbColor();
+        $this->assertTrue($validator->assert('#0'));
```

Comment:
It's a good practice to have 1 assert per test, can you please use data providers instead?